src/app/error.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import Link from 'next/link';
+import { useEffect } from 'react';
+
+export default function Error({
+  error,
+}: {
+  error: Error & { digest?: string };
+}) {
+  useEffect(() => {
```

Comment:
There's not advantage for us to print an error to console, so better to remove useEffect and 'use client' statement. :)